packages/angular/README.md

```diff
@@ -1,6 +1,6 @@
 # @synergy-design-system/angular
 
-This package provides [Angular](https://angular.io/) wrappers for [Synergy Web Components](https://github.com/synergy-design-system/synergy-design-system/tree/main/packages/components).
+[This package](https://github.com/synergy-design-system/synergy-design-system/tree/main/packages/angular) provides [Angular](https://angular.io/) wrappers for [Synergy Web Components](https://github.com/synergy-design-system/synergy-design-system/tree/main/packages/components).
```

Comment:
I think it's more important to link to Storybook in the README or ideally in the package.json (homepage).

![CleanShot 2023-12-11 at 09 18 20@2x](https://github.com/synergy-design-system/synergy-design-system/assets/26542182/ec7a38e8-300c-47e1-bb4c-136f45b72a1a)

Reply:
changed it